README.md

```diff
@@ -43,6 +43,38 @@ docker-compose down --remove-orphans -v
 docker-compose up --build    
 ```
 
+## Using Windows WSL
```

Comment:
:broken_heart:

Reply:
Don't cry, it is only for development purposes...